### PR TITLE
Fixed showing "developing" for shipped features

### PR DESCRIPTION
**Pull request by phistuck**
Fixed https://github.com/mozilla/platform-status/issues/484 by comparing to the beta version instead of the no-longer-exists Aurora version. Also removed references to the developer edition (since it is the same as beta) and made the nightly version accurate as it is now exposed in the JSON.

### CODE REVIEW

**Comment by phistuck** [comment]
This is the mentioned JSON -
https://svn.mozilla.org/libs/product-details/json/firefox_versions.json
The Aurora version is empty, which caused the issue.

File: engine/firefoxVersionParser.js [hunk 1]
```diff
@@ -14,8 +14,7 @@ export default class FirefoxVersionParser {
         this.results = {
           stable: this.majorVersion(results.LATEST_FIREFOX_VERSION),
           beta: this.majorVersion(results.LATEST_FIREFOX_DEVEL_VERSION),
-          aurora: this.majorVersion(results.FIREFOX_AURORA),
-          nightly: this.majorVersion(results.FIREFOX_AURORA) + 1, // File doesn't have nightly.
+          nightly: this.majorVersion(results.FIREFOX_NIGHTLY), // File doesn't have nightly.
```
Comment (marco-c) [inline engine/firefoxVersionParser.js]: Nit: can you remove the now outdated comment ("File doesn't have nightly")?

**Comment by marco-c** [comment]
Thanks for the investigation and the PR!

File: engine/firefoxVersionParser.js (hunk 1 above)
Comment (phistuck) [reply engine/firefoxVersionParser.js]: Of course. That is what I get for talking to people while I commit a change. ;)

**Pull request merged**